lms/templates/main.html

```diff
@@ -86,13 +86,12 @@
   <%block name="headextra"/>
 
 <%
-  if theme_enabled() and not is_microsite():
-    header_extra_file = 'theme-head-extra.html'
-    header_file = 'theme-header.html'
-    google_analytics_file = 'theme-google-analytics.html'
-
-    style_overrides_file = None
+  style_overrides_file = microsite.get_value('css_overrides_file')
 
+  if theme_enabled():
+    header_extra_file = microsite.get_template_path('theme-head-extra.html')
+    header_file = microsite.get_template_path('theme-header.html')
+    google_analytics_file = microsite.get_template_path('theme-google-analytics.html')
```

Comment:
In #8271, we're planning to change the names of these files. You can just use `header-extra.html`, and `google_analytics.html` in your theme directory, and the template system should include them. As a result, you can roll the contents of `theme-head-extra.html` into `header-extra.html`.